setup.py

```diff
@@ -203,6 +203,7 @@
             'inspire_cache = inspirehep.modules.cache.ext:INSPIRECache',
             'inspire_search = inspirehep.modules.search:INSPIRESearch',
             'inspire_workflows = inspirehep.modules.workflows:INSPIREWorkflows',
+            'invenio_collections = invenio_collections:InvenioCollections',
```

Comment:
Yup. To me it's not obvious why this would make a difference, so if we have a public error on Travis we can see what's going on.